src/pages/Proxies.tsx

```diff
@@ -201,6 +221,26 @@ export default () => {
                               }}
                             />
                           </Show>
+                          <Show
+                            when={
+                              proxyGroup.icon &&
+                              proxyGroup.icon.startsWith('data:image/svg+xml')
+                            }
+                          >
+                            <div
+                              style={{
+                                'mask-image': `url("${encodeSvg(proxyGroup.icon!)}")`,
+                                '-webkitmask-image': `url("${encodeSvg(proxyGroup.icon!)}")`,
```

Comment:
First, you might want to use `-webkit-mask-image`? Second, I couldn't find this property anywhere.

<img width="328" alt="image" src="https://github.com/user-attachments/assets/f36f631b-d881-4092-82d1-bfaa24bfff10">

Reply:
it's incorrect CSS properties,delete it